shared/appengine.go

```diff
@@ -161,15 +159,7 @@ func (a appEngineAPIImpl) GetVersionedHostname() string {
 }
 
 func (a appEngineAPIImpl) GetServiceHostname(service string) string {
```

Comment:
I guess we had no tests for this then? Not sure if its worth adding much now that the method is just a simple `Sprintf` wrapper though

Reply:
Right, we don't actually have a test for this whole module, which is supposed to be a thin wrapper of appengine APIs.